### ISSUE TITLE
Remove GET_ACCOUNTS permission

### ISSUE THREAD

**Issue by tasomaniac** [gdg-x/frisbee#678]
With the newest Google Auth APIs we shouldn't need GET_ACCOUNTS permission. I'm opening this issue not to forget removal of the following permissions.

```xml
  <uses-permission android:name="android.permission.USE_CREDENTIALS" />
  <uses-permission android:name="android.permission.GET_ACCOUNTS" />
```

**Comment by tasomaniac** [comment]
Blocked y #616

**Comment by tasomaniac** [comment]
Fixed by #692
